aws/resource_aws_spot_fleet_request.go

```diff
@@ -1135,7 +1135,7 @@ func rootBlockDeviceToSet(
 
 func resourceAwsSpotFleetRequestUpdate(d *schema.ResourceData, meta interface{}) error {
 	// http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_ModifySpotFleetRequest.html
-	conn := meta.(*AWSClient).ec2conn
+	//Value not used: conn := meta.(*AWSClient).ec2conn
```

Comment:
@vnguyen7 instead of keeping commented code in this update function that effectively does nothing, we should completely remove `resourceAwsSpotFleetRequestUpdate` and add `ForceNew: true` to any attribute with `Optional: true` or `Required: true`.